### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by DominiqueMakowski**

### CODE REVIEW

File: R/model_text_parameters.R [hunk 1]
```diff
@@ -235,7 +235,7 @@ model_text_parameters_bayesian <- function(model, parameters, ci = 0.90, rope_fu
   }
 
   if (all(estimate_full != "")) {
-    estimate <- paste0(string_split_matrix(estimate_full, ", ")[, 1], ", ")
+    estimate <- paste0(string_split_matrix(estimate_full, ", ")[1, ], ", ")
```
Comment (strengejacke) [inline R/model_text_parameters.R]: This function was a bit hairy, but I hope it works as intended...

File: R/misc_cite_packages.R [hunk 2]
```diff
@@ -17,20 +18,20 @@ show_packages <- function(session) {
...
+      unlist()
 
     i <- 1
-    while (stringr::str_detect(citation[[1]][i], "To cite ")) {
+    while (stringr::str_detect(citation[i], "To cite ")) {
       i <- i + 1
     }
 
-    citation <- gsub("\\s", "", gsub("SPLIT", "", citation[[1]][i]))
+    citation <- gsub("  ", " ", trimws(gsub("\\s", " ", gsub("SPLIT", "", citation[i])), which="both"))
```
Comment (strengejacke) [inline R/misc_cite_packages.R]: Is `trimws()` needed here? I though `gsub("\\s")` will remove any whitespaces anyway?

```r
gsub("\\s", "", " this is a test!   ")
#> [1] "thisisatest!"
```

Comment (DominiqueMakowski) [reply R/misc_cite_packages.R]: Hum I guess you're right, I have to investigate more

Comment (DominiqueMakowski) [reply R/misc_cite_packages.R]: image

It seems (the second line) that it does not work as intended, hence the trimws 🤔

Comment (strengejacke) [reply R/misc_cite_packages.R]: You're replacing white space with white space! :-D

Comment (DominiqueMakowski) [reply R/misc_cite_packages.R]: haha I don't want to remove ALL the whitespaces (want to keep those within the sentence 😄 )

File: R/model_text_parameters.R (hunk 1 above)
Comment (DominiqueMakowski) [reply R/model_text_parameters.R]: fixed 🤞

File: R/misc_cite_packages.R (hunk 2 above)
Comment (strengejacke) [reply R/misc_cite_packages.R]: Oh I see... I don't have the former code in mind, but that was probably indeed too much removing the trimws()

**Pull request merged**